### ISSUE TITLE
say what is "lat" and "lng" on mouseover

### ISSUE THREAD

**Issue by ralfhauser** [mmmatthew/water-fountains.org#105]
some people when doing issue #35 could get confused on what is what - mouse-over would help

![image](https://user-images.githubusercontent.com/1681677/50357327-f1e22700-0555-11e9-89f9-400d72ea1ada.png)

**Comment by ralfhauser** [comment]
This could be a good test case, how someone other than mmdv would do this

**Comment by mmmatthew** [comment]
I agree that this information should be provided, but not as a secret desktop-only solution. I see two alternatives:
- Add a metadata entry that describes what the property is
- Modify the property name from `coordinates` to `coordinates (lng, lat)`

I think the best would be to do both. However the first requires another step, which is that the front-end reads meta-information from the server on page load. (We don't want to carry repetitive metainfo in all fountain data)

**Comment by ralfhauser** [comment]
please do both (albeit it is not clear to me what your first bullet will look like and where you plan to place it)

**Comment by mmmatthew** [comment]
The description would be placed under the title
![image](https://user-images.githubusercontent.com/10126798/53643035-5b0ba800-3c33-11e9-9593-d29ec0d23f6c.png)
